zebrad/src/components/mempool/downloads.rs

```diff
@@ -442,12 +442,11 @@ where
         self.pending.len()
     }
 
-    /// Check if transaction is already in the state.
+    /// Check if transaction is already in the best chain.
     async fn transaction_in_state(
```

Comment:
Nitpick: Could we rename this method as well?

Reply:
Whoops, yeah, renamed in [`65f39ba`](https://github.com/ZcashFoundation/zebra/pull/5716/commits/65f39ba7be9bd586d3a4762502ad1e56a864acdb).